Added docs for Conversion and Review Tools (#112)
openlawteam/docs: docs/review-tool/README.md

---
meta:
  - name: description

>     content: Creating and publishing high-quality smart legal agreements from scratch is a tough task, but now that you know more about the OpenLaw Markup Language you can help us to automate this process.

Reviewer: -> Language,

With these steps we have a feedback loop that leverages a new flow of information and raises the standards
of the converted files.

> The Review Tool is an important piece of this puzzle and now our community can [sign up](https://conversion.openlaw.io/register)

Reviewer: I don't know if it's a good idea to call it a puzzle, it could give the impression of something being confusing, maybe ecosystem?

  the agreement to review it again.
  :::

## Interactive Labeling

The Review Tool identifies different types of entities and variables. However we often see invalid or partial matches, and
with the interactive labeling tool we can review and fix this sort of information.

### Variable Classification

> The picture bellow shows a partial match of an Organization name, the same might happen for Person names:

Reviewer 2: ```suggestion
The picture below shows a partial match of an Organization name, and the same might happen for Person names:
```

    <img src="./img/review-popup-fix-entity.png" alt="Fix Entity" />
</center>

And set correct entity type to (`Person`), it will be applied to the text and move to the next item if available.

At this stage of the review all classified entities will be converted into OpenLaw Markdown variables and applied to
the background text in the editor.

With the variables updated in the editor the Review Tool starts the next stage automatically: [Variable Renaming](#variable-renaming).

---

### Variable Renaming

After reviewing the entity matches and types it is time to give a proper name to the variables.

> Each one of entities you have classified in the previous step is now a Markup Variable.

Reviewer 2: ```suggestion
Each one of the entities you have classified in the previous step is now a Markup Variable.
```

</center>

`Person`

<center>
    <img src="./img/renaming-popup-person-var.png" alt="Person Var" />
</center>

Just fill out the input text with a proper name and it will replace the highlighted in the original
markdown content at the end of this step. If you need more context to choose a name you can simply look at the editor
in the background which surfaces the current variable in review. Leave it blank to keep the default name.

<center>
    <img src="./img/review-editor-background-var-highlight.png" alt="Highlight Var" />
</center>

After all variables are renamed the tool will move to the next step automatically: [Title Suggestion](#title-suggestion).

---

### Title Suggestion

At this step it asks for a title suggestion to the agreement that is being reviewed.
We often see odd names for these markdown files, so in order to improve that we can fill out the
input text with a suitable name. Leave it blank to keep the default.

<center>
    <img src="./img/title-suggestion-popup.png" alt="Title Suggestion" />
</center>

Hit **Save** and all the changes you have suggested will be applied to the content displayed in the text [editor](#editor).

---

## Editor

### Edit

> The text editor allows you to fix typos, add/fix variables that were not identified at [Variable Classification](#variable-classification) or

Reviewer 2: ```suggestion
The text editor allows you to fix typos, add/fix variables that were not identified at the [Variable Classification](#variable-classification) or
```
